2018-02-28T03:23:51.063Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '0000' in collection with ID 'Participant:org.krow.model.User' does not exist)
    at /home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-28T03:23:51.064Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '0000' in collection with ID 'Participant:org.krow.model.User' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '0000' in collection with ID 'Participant:org.krow.model.User' does not exist)\n    at channel.queryByChaincode.then.catch (/home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
